internal/command/workspace_output_test.go

```diff
@@ -0,0 +1,185 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package command
+
+import (
+	"context"
+	"encoding/json"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/go-tfe"
+	"github.com/hashicorp/tfci/internal/cloud"
+	"github.com/hashicorp/tfci/internal/environment"
+	"github.com/mitchellh/cli"
+)
+
+type WorkspaceOutputReader struct {
+	svo *tfe.StateVersionOutputsList
+}
+
+func (w *WorkspaceOutputReader) ReadStateOutputs(_ context.Context, orgName string, wName string) (*tfe.StateVersionOutputsList, error) {
+	return w.svo, nil
+}
+
+type testWorkspaceOutputCommandOpts struct {
+	items []*tfe.StateVersionOutput
+}
+
+func testWorkspaceOutputCommand(t *testing.T, opts *testWorkspaceOutputCommandOpts) (*cli.MockUi, *WorkspaceOutputCommand) {
+	t.Helper()
+
+	if opts.items == nil || len(opts.items) < 1 {
+		opts.items = []*tfe.StateVersionOutput{{
+			Name:      "test",
+			Value:     "",
+			Sensitive: false,
+		}}
+	}
+
+	cloudMockService := &cloud.Cloud{
+		WorkspaceService: &WorkspaceOutputReader{
+			svo: &tfe.StateVersionOutputsList{
+				Items: opts.items,
+			},
+		},
+	}
+	ui := cli.NewMockUi()
+	meta := NewMeta(cloudMockService)
+	meta.Ui = ui
+	meta.Env = &environment.CI{}
+
+	return ui, &WorkspaceOutputCommand{Meta: meta}
+}
+
+func TestWorkspaceOutputListCommand_Output(t *testing.T) {
+	testCases := []struct {
+		name    string
+		args    []string
+		svoList []*tfe.StateVersionOutput
+	}{
+		{
+			name: "standard-values",
+			args: []string{"--workspace=my-workspace"},
+			svoList: []*tfe.StateVersionOutput{{
+				Name:  "image_id",
```

Comment:
nit: indent.

I know it's shorter, but seeing the closing braces at the same indent looks odd to me. I'd prefer we not collapse the array items like this.